docs/conf.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
```

Comment:
I have thought a bit more about it, and actually see the value in providing intersphinx and a buildable set of docs as a starting point. Not only because it might help users, but also because then there's a clear distinction between `.md`, which is meta-information about this repo, and `.rst`, which is the foundation for documentation of repositories that start off from this one.